PublishParasoftResults/ParaReportPublishService.ts

```diff
@@ -205,13 +205,8 @@ export class ParaReportPublishService {
     run = async (): Promise<void> => {
         // Check if there are multiple "Publish Parasoft Results" tasks in the pipeline to prevent confusion
         const publishTaskExists = tl.getVariable('PF.PublishParasoftResultsExists');
```

Comment:
I think we can change the variable name to `PF.PublishTaskExists` to be consistent with the `const`.